src/commcare_cloud/ansible/roles/keepalived/tasks/main.yml

```diff
@@ -15,6 +15,17 @@
   tags: keepalived
   notify: restart keepalived
 
+- name: Create redis master check script 
```

Comment:
shouldn't this also be in the redis role?

Reply:
Moved.

Reply:
Can't notify from playbook. where should i keep this ?

Reply:
Why does it need to notify for changes to this script?

Reply:
It's Needed for first run 
```
Jan 22 05:06:59 MUMGCCWCDPRDRDV05 Keepalived_vrrp[121144]: Script /etc/keepalived/redis_is_master cannot be accessed - No such file or directory
Jan 22 05:06:59 MUMGCCWCDPRDRDV05 Keepalived_vrrp[121144]: Disabling track script chk_service since not found/accessible
```
if it doesn't find the file at start time. it will disable tracking of service.  So we need to restart at the time of creating this file.

Reply:
we have to keep this task in one of the role either the keepalived or redis .

Reply:
OK